app/latigo/metadata_api/client.py

```diff
@@ -68,3 +67,18 @@ def post(self, *args, **kwargs) -> Response:
         """Make POST call the the API. Standard params are allowed."""
         res = self.session.post(*args, **kwargs)
         return res
+
+    def send_time_series_id_metadata(self, time_series_ids_metadata: TimeSeriesIdMetadata) -> Response:
+        """Save time series id metadata to the Metadata API after storing it in the Time Series API."""
+        url = f"{self.base_url}/models"
+
+        res = self.post(url=url, data=self._dump_data(asdict(time_series_ids_metadata)))
+        return res
+
+    @staticmethod
+    def _dump_data(data: Any) -> str:
```

Comment:
```suggestion
    def _dump_data(data: Dict) -> str:
```
According to the docstring.